lib/utils/session_vars.js

```diff
@@ -197,6 +198,36 @@ session_vars.save_artifacts_path_name = function ( path_name ) {
   return file;
 };  // Ends save_artifacts_path_name()
 
+
+session_vars.save_install_type = function (install_type) {
```

Comment:
From what I can see, this can possibly be taken care of with the `_ORIGIN` env var. Is there a reason it needs to be saved into the runtime config instead that I'm not seeing?

Reply:
I see now that `install_type` is different in definition than `_ORIGIN` conceptually. `install_type` is if it was installed in the playground vs. on the server. That is a more clear description. I'm wondering if it's still enough to just infer it from `_ORIGIN` or if we really want to introduce a new action step-crossing config variable. I'll restore the previous code, but I think it's worth discussing.

Reply:
1) I think of `_ORIGIN` and `install_type` as separate things: `_ORIGIN` should only deal with where Kiln JS is running, and `install_type` is independent of that. I can easily see GitHub actions or local runs using either installed interviews or playground interviews. I agree it's less likely to run KilnInThePlayground on an installed interview, but...

2) I still hold that `_ORIGIN` is a temporary solution, and the problem it solves is better and more flexibly solved by command args for parameters, so I've been avoiding building more functionality on top of it.

Reply:
Deep dive discussion: Since the two vars could be used in combination someday, we'll keep them separate for now. We'll also add the `install_type` interview url to the arguments that will be passed in through the command line when we do the same for the file names (avoiding `_ORIGIN`, though we do have to check if that env var is used for something else too), so for now we'll keep things as they are since they'll be changed anyway.

Reply:
Conclusion: We used `install_method` and kept `_ORIGIN` separate.